src/pages/sykmelding/__test__/arbeidstaker.test.tsx

```diff
@@ -78,12 +78,12 @@ describe('Arbeidstaker', () => {
         render(<SykmeldingPage />, { ...renderOptions, history });
 
         userEvent.click(await screen.findByRole('radio', { name: 'Ja' }));
-        userEvent.click(await screen.findByRole('radio', { name: 'arbeidstaker' }));
+        userEvent.click(await screen.findByRole('radio', { name: 'ansatt' }));
         userEvent.click(await screen.findByRole('radio', { name: arbeidsgivereMock[0].navn }));
         const naermesteLederFieldset = screen.getByText(/som skal følge deg opp/i).closest('fieldset');
         userEvent.click(within(naermesteLederFieldset!).getByRole('radio', { name: 'Ja' }));
 
-        expect(await screen.findByRole('article', { name: 'Dette får arbeidsgiveren din se' })).toBeInTheDocument();
+        expect(screen.queryByText('Dette vises til arbeidsgiveren din')).toBeInTheDocument();
```

Comment:
En annen ting er at man burde strebe etter å bruke `*ByRole` så ofte som mulig, da den tvinger oss til å tenge litt mer på UU. For ikke-interaktive elementer er det greit å bruke `*byText` men man kan fortsatt gjøre et forsøk på å tenke på UU. :smile: 

https://testing-library.com/docs/queries/about/#priority

> getByRole: This can be used to query every element that is exposed in the accessibility tree. With the name option you can filter the returned elements by their accessible name.